.upstream-tests/test/cuda/atomics/atomic.ext/atomic_helpers.h

```diff
@@ -62,6 +62,20 @@ struct TestEachIntegralType {
     }
 };
 
+template < template <class, template<typename, typename> class, cuda::thread_scope> class TestFunctor,
+    template<typename, typename> class Selector, cuda::thread_scope Scope
+#if !defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 600
+    = cuda::thread_scope_system
+#endif
+>
+struct TestEachFloatingPointType {
+    __host__ __device__
+    void operator()() const {
+        TestFunctor<float, Selector, Scope>()();
+        TestFunctor<double, Selector, Scope>()();
```

Comment:
Should we also have a host-only call to `TestFunctor<long double, Selector, Scope>()()` here?